From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?P=C5=99emek=20Vysok=C3=BD?= <premek.vysoky@microsoft.com>
Date: Tue, 4 Feb 2025 18:10:48 +0100
Subject: [PATCH] Remove extra spaces in Publishing.props (#6251)

This might help with the VMR sync problems we have in https://github.com/dotnet/sdk/pull/46361
Backport: https://github.com/NuGet/NuGet.Client/pull/6251
---
 eng/Publishing.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/Publishing.props b/eng/Publishing.props
index d5604fbf8..5fe1ceb1a 100644
--- a/eng/Publishing.props
+++ b/eng/Publishing.props
@@ -4,5 +4,5 @@
     <Artifact Include="$(ArtifactsDir)nupkgs/*.nupkg"
               PublishFlatContainer="false" />
   </ItemGroup>
-  
+
 </Project>
